src/rc/start-stop-daemon.c

```diff
@@ -103,6 +108,8 @@ const struct option longopts[] = {
 	{ "stdout-logger",1, NULL, '3'},
 	{ "stderr-logger",1, NULL, '4'},
 	{ "progress",     0, NULL, 'P'},
+	{ "scheduler",    1, NULL, 0x81},
```

Comment:
i just updated the code to use enums so we don't have to maintain constants anymore